runtime/engine/src/main/resources/META-INF/services/jakarta.json.spi.JsonProvider

```diff
@@ -0,0 +1 @@
+org.leadpony.joy.yaml.YamlProvider
```

Comment:
This file is already present in `joy-yaml` JAR, and not required by `engine` project otherwise.

If dependencies are being picked up correctly by `shade` plugin, shouldn't `joy-yaml` dependency cause this to be included in the shaded JAR without having to explicitly put it into our sources?